packages/migrate/src/commands/DbExecute.ts

```diff
@@ -164,13 +164,14 @@ See \`${green(getCommandWithExecutor('prisma db execute -h'))}\``,
     else {
       // validate that schema file exists
       // throws an error if it doesn't
-      const schemaPath = await getSchemaPath(args['--schema'])
+      const schemaPathResult = await getSchemaPath(args['--schema'])
 
       // Execute command(s) to url from schema
       datasourceType = {
         tag: 'schema',
-        // if schemaPath is undefined, getSchemaPath will error
-        schema: schemaPath!,
+        // TODO: `schema` should have type `Datamodel`, see:
```

Comment:
I think we wanted to stop using `Datamodel` in reference to prisma schema in a new code.

Reply:
Oh yes, that would be nice to get rid of the old "datamodel" mentions